dist/bnd.bnd

```diff
@@ -38,6 +38,7 @@
             bndtools.jareditor, \
             bndtools.m2e, \
             bndtools.pde, \
+            bndtools.ecf, \
```

Comment:
I had a look at the code - you haven't actually created a `bndtools.ecf` project in the workspace, have you? Only the feature?

The build is failing because `-dependson` doesn't list Eclipse features - it contains the list of projects in the Bnd workspace that the feature/distribution depends on. There is a `bndtools.pde` and a `bndtools.m2e` project, that is why they are included in `-dependson` - but if the feature you're adding doesn't actually contain a corresponding bundle called `bndtools.ecf`, then you won't need to include those in the `-dependson`. As far as I can see there is no Bnd project in this commit called `bndtools.ecf`.

I suspect if you remove this (and its partner-in-crime below), it will work.